src/utils/latency_tracer.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2017-present, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+#include "latency_tracer.h"
+
+#include <dsn/service_api_c.h>
+#include <dsn/dist/fmt_logging.h>
+#include <dsn/utility/flags.h>
+
+namespace dsn {
+namespace utils {
+
+DSN_DEFINE_bool("replication", enable_latency_tracer, true, "whether enable the latency tracer");
```

Comment:
```suggestion
DSN_DEFINE_bool("replication", enable_latency_tracer, false, "whether enable the latency tracer");
```

Tracing should be off by default.